src/applications/simple-forms/shared/tests/e2e/helpers.js

```diff
@@ -1,3 +1,19 @@
+export const getPagePaths = formConfig => {
```

Comment:
This'll save us from manually writing out page-paths for v3-web-component unit-tests.